doc/mgmt/Management Framework.md

```diff
@@ -1295,15 +1668,19 @@ The function prototypes for external transformer callbacks are defined in the fo
 
 ```go
 type XfmrParams struct {
-	d *db.DB
-	dbs [db.MaxDB]*db.DB
-	curDb db.DBNum
-	ygRoot *ygot.GoStruct
-	uri string
-	oper int
-	key string
-	dbDataMap *map[db.DBNum]map[string]map[string]db.Value
-	param interface{}
+        d *db.DB
+        dbs [db.MaxDB]*db.DB
+        curDb db.DBNum
+        ygRoot *ygot.GoStruct
+        uri string
+        requestUri string //original uri using which a curl/NBI request is made
+        oper int
+        key string
+        dbDataMap *map[db.DBNum]map[string]map[string]db.Value
+        subOpDataMap map[int]*RedisDbMap // used to add an in-flight data with a sub-op
+        param interface{}
+        txCache *sync.Map
```

Comment:
Can you shed light on what is txCache and how does it use and what is the use case we are trying to solve here?

Reply:
The txCache (transaction cache) is a placeholder to cache data between transformer callbacks. 
It can be used for the case that a callback need some optimization by caching data and passing to other callbacks for subsequent use of the same data retrieved from the txCache of the inParams. The txCache is bound to the given transaction scope.